From 2f01f37f4eeaa2864ae926fc8ceaef9cc9dbc847 Mon Sep 17 00:00:00 2001
From: naz664 <nazimnavas145@gmail.com>
Date: Sun, 22 Jan 2023 03:56:52 +0000
Subject: [PATCH 2/5] Revert udfps: Dimming implication on arrow config

This revert the udfps dimming commit on file arrow_config.xml which missed on previous revert commit. Because it lead to conflict while applying it next time.
---
 packages/SystemUI/res/values/arrow_config.xml | 36 -------------------
 1 file changed, 36 deletions(-)

diff --git a/packages/SystemUI/res/values/arrow_config.xml b/packages/SystemUI/res/values/arrow_config.xml
index 288ae3980988..9f0a477ba987 100644
--- a/packages/SystemUI/res/values/arrow_config.xml
+++ b/packages/SystemUI/res/values/arrow_config.xml
@@ -39,42 +39,6 @@
     <!-- Color of the UDFPS pressed view -->
     <color name="config_udfpsColor">#ffffffff</color>
 
-    <!-- Flag to enable framework dimming for udfps -->
-    <bool name="config_udfpsFrameworkDimming">false</bool>
-
-    <!-- Array of brightness-alpha lut for framework dimming -->
-    <string-array name="config_udfpsDimmingBrightnessAlphaArray" translatable="false">
-          <!-- Example:
-          <item>0,255</item>
-          <item>1,234</item>
-          <item>3,227</item>
-          <item>8,208</item>
-          <item>16,192</item>
-          <item>27,176</item>
-          <item>41,160</item>
-          <item>61,144</item>
-          <item>80,128</item>
-          <item>104,112</item>
-          <item>130,96</item>
-          <item>158,80</item>
-          <item>188,64</item>
-          <item>221,48</item>
-          <item>250,36</item>
-          <item>255,33</item>
-          -->
-    </string-array>
-
-    <!-- Brightness range min for udfps dimming -->
-    <integer name="config_udfpsDimmingBrightnessMin">0</integer>
-
-    <!-- Brightness range max for udfps dimming -->
-    <integer name="config_udfpsDimmingBrightnessMax">0</integer>
-
-    <!-- The amount of delay to add when disabling the dimming.
-         This is used to prevent flickers due to the dimming being disabled
-         before the screen has had chance to switch out of HBM mode -->
-     <integer name="config_udfpsDimmingDisableDelay">0</integer>
-
     <!-- Whether usage of the proximity sensor during doze is supported -->
     <bool name="doze_proximity_sensor_supported">true</bool>
 
-- 
2.25.1

